### PR TITLE
Store the HTML and TEXT parts of a stored message in the mock server

### PR DESCRIPTION
**Pull request by itsdax**
Mock Server currently drops the HTML and Text portions of the stored email. This allows users to place assertions on the contents of an email. 

For example, my service wants to assert that it has successfully sent an email with a login verification code. This provides the capability to do so.

PTAL - Thanks!

### CODE REVIEW

**Comment by vtopc** [comment]
> This allows users to place assertions on the contents of an email.

In which way? Those new fields are private.

**Comment by vtopc** [comment]
@itsdax, hi. 
The mockServer was intended to be a mock for testing the github.com/mailgun/mailgun-go itself.
What are the use cases that you need to be handled with these changes?

**Comment by vtopc** [comment]
Closing as abandoned. Feel free to reopen it again.

**Comment by itsdax** [comment]
Hey sorry for the slow response - it's been a while since I've created this PR.

While I understand this repo is primarily for testing purposes of the repo itself, we've found the test suite quite extendable for generic upstream testing use cases. Specifically, we're using it to create a mock Mailgun server with minimal effort for our integration tests. Let me know if this is convincing enough, or if it's a bit of a stretch to include for this testing suite.

File: mock_messages.go [hunk 1]
```diff
@@ -49,6 +52,10 @@ func (ms *mockServer) createMessages(w http.ResponseWriter, r *http.Request) {
 		stored.Flags = events.Flags{
 			IsTestMode: false,
 		}
+
+		ms.storedMessageHtml[id] = r.FormValue("html")
+		ms.storedMessageText[id] = r.FormValue("text")
+
 		ms.mutex.Lock()
```
Comment (vtopc) [inline mock_messages.go:59]: Should be guarded by mutex too

**Comment by vtopc** [comment]
Closing as abandoned. Please feel free to reopen it if you need to.

**Pull request closed without merging**